packages/flutter_lints/pubspec.yaml

```diff
@@ -2,13 +2,13 @@ name: flutter_lints
 description: Recommended lints for Flutter apps, packages, and plugins to encourage good coding practices.
 repository: https://github.com/flutter/packages/tree/main/packages/flutter_lints
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+flutter_lints%22
-version: 5.0.0
+version: 6.0.0
 
 environment:
   sdk: ^3.5.0
```

Comment:
According to the CI failures, this is no longer correct; `lints` 6.0 requires 3.8, so this needs to require 3.8 as well.

That also means this PR can't land until after the next stable release, as our policy is to support stable.